api/src/components/integration/controller.ts

```diff
@@ -48,3 +48,16 @@ export const incrementalAriIngest = async (
         return response.json(500, { message: 'Unknown server error.' });
     }
 };
+
+export const triggerECSTask = async (event: I.APIRequest): Promise<I.JSONResponse> => {
+    const apiKey = event.queryStringParameters?.apiKey;
```

Comment:
Can this be done with middleware instead? Keep it consistent with other routes

Reply:
Yes, that's probably better - just changed this here and a couple of other places